lib/components/Datepicker/Datepicker.tsx

```diff
@@ -1,14 +1,28 @@
-import { DatePicker as ArkDatePicker, Portal, UseDatePickerContext } from '@ark-ui/react';
+import { DatePicker as ArkDatePicker, parseDate, Portal, UseDatePickerContext } from '@ark-ui/react';
 import React from 'react';
 import { MdArrowBack, MdArrowForward, MdCalendarMonth } from 'react-icons/md';
 import { cx } from '../../cva';
+import { tidyClasses as tc } from '../../utils';
 
 const MIN_YEAR = 1900;
 const MAX_YEAR = 2100;
 const isInvalidYear = (year: number) => year < MIN_YEAR || year >= MAX_YEAR;
 
-const tableCellClasses =
-  'ds-m-3 ds-text-body-sm ds-font-arial ds-size-[28px] sm:ds-size-[32px] hover:ds-underline focus:ds-left-auto ds-capitalize data-[selected]:ds-bg-secondary-1-50 data-[selected]:ds-rounded-full';
+const tableCellClasses = tc([
+  'ds-flex',
```

Comment:
For some reason the numbers in datepicker UI became misaligned, so flex needed to be added.